frontend/src/layouts/ProjectLayout/ProjectLayout.tsx

```diff
@@ -134,18 +134,48 @@ export const ProjectLayout = () => {
                         </Link>
                       )}
                       {isSSH && (
-                        <Link
-                          to={`/${ProjectType.SSH}/$projectId/overview` as const}
-                          params={{
-                            projectId: currentWorkspace.id
-                          }}
-                        >
-                          {({ isActive }) => (
-                            <MenuItem isSelected={isActive} icon="lock-closed">
-                              Overview
-                            </MenuItem>
-                          )}
-                        </Link>
+                        <>
+                          <Link
+                            to={`/${ProjectType.SSH}/$projectId/overview` as const}
+                            params={{
+                              projectId: currentWorkspace.id
+                            }}
+                          >
+                            {({ isActive }) => (
+                              <MenuItem isSelected={isActive} icon="server">
+                                Hosts
+                              </MenuItem>
+                            )}
+                          </Link>
+                          {/* <Link
+                            to={`/${ProjectType.SSH}/$projectId/certificates` as const}
+                            params={{
+                              projectId: currentWorkspace.id
+                            }}
+                          >
+                            {({ isActive }) => (
+                              <MenuItem isSelected={isActive} icon="certificate" iconMode="reverse">
+                                Certificates
+                              </MenuItem>
+                            )}
+                          </Link> */}
+                          {/* <Link
+                            to={`/${ProjectType.SSH}/$projectId/cas` as const}
+                            params={{
+                              projectId: currentWorkspace.id
+                            }}
+                          >
+                            {({ isActive }) => (
+                              <MenuItem
+                                isSelected={isActive}
+                                icon="certificate-authority"
+                                iconMode="reverse"
+                              >
+                                Certificate Authorities
+                              </MenuItem>
+                            )}
+                          </Link> */}
+                        </>
```

Comment:
sayonara!

Reply:
so are we keeping this hidden? I see that you made some changes to the SSH CAs page so just confirming

Reply:
Yeah I'm keeping it hidden for now; there's a chance that we'll have to bring some parts of it back depending on actual usage.

I foresee it's possible that users will want to create and connect their own SSH CAs to hosts registered in Infisical.